src/components/UploadSource/UploadSource.md

```diff
@@ -0,0 +1,147 @@
+This component helps to upload bible books
+
+### Upload Source
+
+```js
+import UploadSource from './UploadSource';
+import Button from '@material-ui/core/Button';
+import Grid from '@material-ui/core/Grid';
+import { API } from '../../store/api';
+import SnackBar from '../SnackBar/SnackBar.js';
+
+const [selectedFiles, setSelectedFiles] = React.useState('');
+const [responseStatus, setResponseStatus] = React.useState([]);
+
+const handleClose = () => {
+  setResponseStatus([false]);
+};
+
+const [projectData, setProjectData] = React.useState({
```

Comment:
This object is not needed as a state variable, projectId is all that is used 
This object can be replace by const projectId=100000;
and projectId can be used directly in the code below.
Ideally a project dropdown would be ideal but this is fine as we are in a hurry for the june release|